[PTFE-583] ✨ Prevent creation of integration branches (#129)
scality/bert-e: bert_e/templates/request_integration_branches.md

{% extends "message.md" %}

{% block title -%}
Request integration branches
{% endblock %}

{% block message %}
Waiting for integration branch creation to be requested by the user.

> To request integration branches, please comment on this pull request with the following command:

Reviewer: What about the fact we wanted other comments to be able to set this value to true?

Author: This message is displayed for people who need to type this command. In the case where someone writes `approve` at the beginning of their PR, they will instead receive a predefined message template that says: 'Integration branches have been created, congratulations! Here they are: ... blablabla.' They will never see this specific template, so I don't think it's necessary to mention it.

{% extends "message.md" %}

{% block title -%}
Request integration branches
{% endblock %}

{% block message %}
Waiting for integration branch creation to be requested by the user.

To request integration branches, please comment on this pull request with the following command:

```
/create_integration_branches
```

> Alternatively, there's another way to accomplish this. Simply tag this PR with `/approve` or
> `/create_pull_requests`, and it will automatically create the integration branches

Reviewer 2: ```suggestion
Alternatively, the `/approve` and `/create_pull_requests` commands will automatically
create the integration branches.
```
